src/Blog/View/posts.twig

```diff
@@ -12,11 +22,15 @@
 	<body class="top-wave">
 	{% endblock %}
 	<section class="container">
-		<h1>Blog</h1>
 		<ul>
 			{% for post in posts %}
-				<li>
-					<a href="/blog/{{ post.id }}">{{ post.title }}</a>
+				<li class="u-margin--medium clearfix">
+					<a class="post-box__img" href="/blog/{{ post.id }}"> <img src="{{post.url}}" alt= "{{post.title}}"/> </a>
+					<div class="post-box__description clearfix">
+					<a class="post-description__header" href="/blog/{{ post.id }}"> <h2 class="post-description__title"> {{post.title}}</h2> <span class="post-descritpion__date"> {{post.date}} </span> </a>
```

Comment:
Delete space between tags and between context and tag.
Element a is nested in div, add tab before a